src/components/Learn-Components/what-await-section/Features-carousel/index.js

```diff
@@ -0,0 +1,94 @@
+import React, {useState} from "react";
```

Comment:
Why this file is being added and where is this used?

Reply:
And it's still not responsive, is what I see 😕

Reply:
<img width="401" alt="Screenshot 2021-07-14 at 11 50 22 AM" src="https://user-images.githubusercontent.com/75248557/125572916-2e75008f-3336-4aae-b6e8-d1010a641692.png">

@Nikhil-Ladha , it is responsive 🤔 , can you check again ?

Reply:
I think it's not responsive as I can see at 768px & above. I am not sure about the background color here .. as below 768px this color is getting disappeared .

![image](https://user-images.githubusercontent.com/44670961/125574624-1a485d3b-4532-4b89-a023-c7580a98be26.png)